src/main/java/org/radarcns/managementportal/Project.java

```diff
@@ -0,0 +1,163 @@
+package org.radarcns.managementportal;
+
+/*
+ * Copyright 2017 King's College London
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.JsonFactory;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Optional;
+
+import okhttp3.Response;
+
+/**
+ * Java class defining a RADAR Management Portal Project.
+ */
+public class Project {
+
+    @JsonProperty("id")
+    private final Integer id;
+    @JsonProperty("projectName")
+    private final String projectName;
+    @JsonProperty("organization")
+    private final String organization;
+    @JsonProperty("location")
+    private final String location;
+    @JsonProperty("attributes")
+    private final List<Tag> attributes;
```

Comment:
Might as well be `Map<String, String>`

Reply:
Yes I made the changes but not yet pushed.

Reply:
I think the Project still uses the array [] values for attributes, So this should be fine. Only change was in the subject attributes which now uses object {} .